raiden/api/python.py

```diff
@@ -1124,10 +1143,16 @@ def get_raiden_events_payment_history_with_timestamps(
             logical_and=False,
         )
 
+        chain_state = chain_state = views.state_from_raiden(self.raiden)
```

Comment:
One assignment should be enough

Reply:
LOL WTF how come no linter complained about this